tests/zfs-tests/cmd/stride_dd.c

```diff
@@ -195,11 +195,13 @@ main(int argc, char *argv[])
 		}
 
 		if (stride > 1) {
-			if (lseek(ifd, (stride - 1) * bsize, SEEK_CUR) == -1) {
+			if (lseek(ifd, (stride - 1ULL) * bsize,
```

Comment:
Thinking about it, a similar treatment could be given to `stride` by making its type `off_t`, but this is fine too.

Reply:
Added cast to off_t